### PR TITLE
Shruti assignment9 avgsum

### PR DESCRIPTION
**Pull request by ShrutiDJun20**
Please Review

### CODE REVIEW

File: src/shrutiDJun20/AverageSum.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+public class AverageSum {
+	
+	 double summation(double x,double y,double z ){
+		double sum = x+y+z;
+		System.out.println("Sum is "+ sum);
+		return sum;
+	}
+	 
+	 double avg(double x,double y,double z){
+		 AverageSum sumobject = new AverageSum();
```
Comment (KrishnaKTechnocredits) [inline src/shrutiDJun20/AverageSum.java]: No need to create new object. non static to non static method can be called directly.

File: src/shrutiDJun20/AverageSum.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+			else
+				return "Grade B";
+						
+		}
+		
+		public static void main(String[] args) {
+			AverageSum averageSum = new AverageSum();
+			double average= averageSum.avg(5, 5, 5);
+			isAnswerAboveExpecation(average);
+			isEligible(average);
```
Comment (KrishnaKTechnocredits) [inline src/shrutiDJun20/AverageSum.java]: You Should print in this method. isAnswerAboveExpecation and isEligible

File: src/shrutiDJun20/AverageSum.java (hunk 1 above)
Comment (ShrutiDJun20) [reply src/shrutiDJun20/AverageSum.java]: ok..changes done

File: src/shrutiDJun20/AverageSum.java (hunk 2 above)
Comment (ShrutiDJun20) [reply src/shrutiDJun20/AverageSum.java]: done as per comment

File: src/shrutiDJun20/AverageSum.java [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+		 if(avg>50)
+			 return true;
+		 else
+			 return false;
+		 }
+		
+	   static boolean isEligible(double avg){
+			if(avg>50 || avg%2==0)
+			return true;
+
```
Comment (KrishnaKTechnocredits) [inline src/shrutiDJun20/AverageSum.java:48]: next time remove extra space.

File: src/shrutiDJun20/AverageSum.java [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+public class AverageSum {
+	
+	 double summation(double x,double y,double z ){
+		double sum = x+y+z;
+		System.out.println("Sum is "+ sum);
+		return sum;
+	}
+	 
+	 double avg(double x,double y,double z){
+
```
Comment (KrishnaKTechnocredits) [inline src/shrutiDJun20/AverageSum.java:32]: next time remove extra space.

File: src/shrutiDJun20/AverageSum.java [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+			
+			else
+				return false;
+			
+		}
+		
+		static String getYourGrade(double avg){
+			if(avg>80){
+				return "Grade A";
+			}
```
Comment (KrishnaKTechnocredits) [inline src/shrutiDJun20/AverageSum.java:57]: remove curly braces, if only one statement is there in if condition.

File: src/shrutiDJun20/AverageSum.java [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+				return "Grade B";
+						
+		}
+		
+		public static void main(String[] args) {
+			AverageSum averageSum = new AverageSum();
+			double average= averageSum.avg(110, 100, 60.00);
+			if(isAnswerAboveExpecation(average)==true)
+				 System.out.println("Result is above expectation");
+				 else
```
Comment (KrishnaKTechnocredits) [inline src/shrutiDJun20/AverageSum.java:68]: work on indentation, else should be at the same level of If condition (66 line).

**Review by KrishnaKTechnocredits — APPROVED** [review]
Remove blank line, only two methods should be separated with 1 blank line & indentation should be more clear. Please refer #151 (Aashtha's code)

**Pull request closed without merging**